getting-started/first-steps-designing.md

```diff
@@ -82,7 +82,7 @@ So, first, let's connect to the data and, then, you'll create the table to prese
 1. On the screen that loads, fill in the __Select Statement__ field with the query below. The query will extract only the first 14 employees and they will be listed in ascending order according to their `id`, that is, the employee with an `id=1` will be the first one, the employee with an `id=2` will come second, and so on.
 
 	````SQL
-SELECT
```

Comment:
This is purposely untabbed to appear as expected in the final document - see https://github.com/telerik/reporting-docs/wiki/Specifics-When-Creating-Markdown-Articles-for-Telerik-Reporting#code-snippets-in-lists